<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZkFingerDemo</class>
 <widget class="QMainWindow" name="ZkFingerDemo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ZkFingerDemo</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelView">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>60</y>
      <width>320</width>
      <height>400</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnReg">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>120</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>注册</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnVerify">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>120</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>验证</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClear">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>120</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>清除</string>
    </property>
   </widget>
   <widget class="QLabel" name="logInfo">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>500</y>
      <width>640</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>提示:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btnConnect">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>120</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>连接设备</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
